aea/helpers/fingerprint.py

```diff
@@ -79,11 +79,7 @@ def update_fingerprint(configuration: PackageConfiguration) -> None:
     Update the fingerprint of a package.
 
     :param configuration: the configuration object.
-    :return: None
     """
-    # we don't process agent configurations
-    if isinstance(configuration, AgentConfig):
-        return
```

Comment:
It seems this was a pretty easy fix.

Reply:
But why don't we process agent configuration? 

Please replace current comment, which just explains what the code does and which does not need explaining, to why we don't process it.